bellows/cli/util.py

```diff
@@ -86,11 +87,11 @@ def channel_mask(channels):
 
 
 @asyncio.coroutine
-def setup(dev, cbh=None, configure=True):
+def setup(dev, cbh=None, configure=True, baudrate=None):
```

Comment:
Let's make baudrate required here (and put it together with dev, but it'll have to be once you kill the default value). Also remove it being kwarg-like in all the calls to it.

Reply:
Should there be a default value in the cli, so that the use do not have to supply it, but they can change it.